open-machine-learning-jupyter-book/deep-learning/nlp/text-preprocessing.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>I think there needs to be a paragraph explaining the&nbsp;example code. Why is the Sentiment of "I really enjoyed the movie." is neg?&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/739/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>